docs/actions.md

```diff
@@ -44,7 +44,7 @@ Review the following steps and examples to create your first JavaScript action.
 2. Create an action from the following JavaScript function. For this example, the action is called 'hello'.
 
   ```
```

Comment:
There is also a "shell" syntax highlighting available. I wonder if you should go through and mark all the shell command sections with it?